code/modules/projectiles/ammo_datums.dm

```diff
@@ -2856,6 +2856,27 @@
 /datum/ammo/bullet/shrapnel/jagged/on_hit_mob(mob/M, obj/item/projectile/P)
 	if(isXeno(M))
 		M.Slow(0.4)
+
+/*
+//========
+					CAS 30mm impacters
+//========
+*/
+/datum/ammo/bullet/shrapnel/gau  //for the GAU to have a impact bullet instead of firecrackers
+	name = "30mm Multi-Purpose shell"
+
+	damage = 150 //More damaging, but 2x less shells and low AP
+	penetration = ARMOR_PENETRATION_TIER_2
+	accuracy = HIT_ACCURACY_TIER_MAX
+	max_range = 0
+	shrapnel_chance = 100 //the least of your problems
+
+/datum/ammo/bullet/shrapnel/gau/hv
+	name = "30mm Anti-Tank shell"
+
+	damage = 95 //Standard AP vs standard. (more AP for less damage)
```

Comment:
With the amount of AP it has, drop down to 80.
We can tweak these numbers further down the line after seeing it in active gameplay some more.

Reply:
done